examples/api/lib/material/refresh_indicator/sliver_refresh_indicator.0.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for SliverRefreshIndicator
+
+import 'package:flutter/material.dart';
+
+void main() => const MyApp();
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: SliverRefreshControlExample(),
+    );
+  }
+}
+
+class SliverRefreshControlExample extends StatefulWidget {
+  const SliverRefreshControlExample({super.key});
+
+  @override
+  State<SliverRefreshControlExample> createState() => _SliverRefreshControlExampleState();
+}
+
+class _SliverRefreshControlExampleState extends State<SliverRefreshControlExample> {
+  List<Color> colors = <Color>[
+    Colors.amber,
+    Colors.orange,
+    Colors.pink,
+  ];
+  List<Widget> items = <Widget>[
+    Container(color: Colors.pink, height: 100.0),
+    Container(color: Colors.orange, height: 100.0),
+    Container(color: Colors.amber, height: 100.0),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: CustomScrollView(
+        physics: const BouncingScrollPhysics(
```

Comment:
Is this required for this to work? This would not match native scroll physics for folks that want a native look and feel on Android.

Reply:
Yes, this is required to create a overscrolling space which is not available the the older versions of android, I don't think that flutter dose include the new behavior of native android.

Is this can be considered as an issue?